src/aeb_blake2.erl

```diff
@@ -0,0 +1,149 @@
+%%%=============================================================================
```

Comment:
Erlang Blake2 exists as a separate package... This module should not be needed!

Reply:
Då tar vi bort det.